app/templates/components/build-header.hbs

```diff
@@ -37,6 +37,12 @@
           {{/if}}
         </li>
       {{/if}}
+      <li>
+        <a class="commit-branch-url" href="{{urlGitHubBranch}}" title="Look at the branch on GitHub">
+          {{inline-svg 'images/stroke-icons/icon-github' class="icon"}}
+          <span class="label-align">Branch {{item.branch.name}}</span>
+        </a>
+      </li>
```

Comment:
After some more back and forth clicking I was able to reproduce it once. I'll try and see if I can find something